src/test/deserialization_tests.cpp

```diff
@@ -156,7 +156,10 @@ TEST_P(GRPCPredictRequestNegative, ShouldReturnDeserializationErrorForSetBlobExc
     tensorMap[tensorName]->setPrecision(testedPrecision);
     std::shared_ptr<MockIInferRequestFailingInSetBlob> mInferRequestPtr =
```

Comment:
This variable is unused

Reply:
If we do not mock infer requests anymore, we do not need `MockIInferRequestFailingInSetBlob` as well.